requirements.txt

```diff
@@ -1,5 +1,6 @@
 # needed by celery
 sqlalchemy
+pyYAML<6.0
```

Comment:
I think if we do a search/replace of `yaml.load` with `yaml.safe_load`, we can use the latest pyyaml? I believe safe_load is recommended anyway.